girder/girder.local.conf

```diff
@@ -0,0 +1,4 @@
+
+[global]
+tools.proxy.on: True
```

Comment:
What does this setting do?

Reply:
That lets cherrypy know that it should look for headers (namely X-Forwarded-Host) from the proxy when trying to compute its request URLs.

Reply:
Thanks for the info